packages/account/src/Constants/onfido-phrases.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Localize } from '@deriv/translations';
+import { LocaleConfig } from 'onfido-sdk-ui';
```

Comment:
Please sync this with latest master. `onfido-sdk-ui` lib is removed and the functionality is being used from a CDN. So to use type from the lib, u might have to copy the content and use it. This is done in t[his PR](https://github.com/binary-com/deriv-app/pull/10560/files#diff-c654ae593110ade7c423e31b1a7a5c2309ff4695cecff2c816cc49eef3c79aa5). U can create the same file and copy the content